### PR TITLE
BundleFormAlterTest to use browsertestbase

### PR DESCRIPTION
**Pull request by dakala**
First attempt at issue #305 - Convert BundleFormAlterTest to use the new BrowserTestBase

### CODE REVIEW

**Comment by amitaibu** [comment]
Looks good. Would you mind closing this one and opening a new PR against https://github.com/Gizra/og

**Comment by amitaibu** [comment]
Actually I'll just merge this one, and push upstream.

File: og_ui/tests/src/Functional/BundleFormAlterTest.php [hunk 1]
```diff
@@ -3,12 +3,11 @@
 namespace Drupal\og_ui\Tests;
 
 use Drupal\Core\Form\FormState;
-use Drupal\KernelTests\AssertLegacyTrait;
 use Drupal\node\Entity\NodeType;
 use Drupal\og\Og;
 use Drupal\og_ui\BundleFormAlter;
 use Drupal\simpletest\AssertContentTrait;
-use Drupal\simpletest\BrowserTestBase;
+use \Drupal\Tests\BrowserTestBase;
```
Comment (amitaibu) [inline og_ui/tests/src/Functional/BundleFormAlterTest.php]: need to remove the trailing `\`

**Comment by amitaibu** [comment]
(after the minor comment will be fixed)

**Comment by dakala** [comment]
@amitaibu: Just noticed the earlier comment. Is it better to issue all PRs against Gizra/og? It might be better to change my upstream to that repo.

**Comment by amitaibu** [comment]
Yeah the Gizra/og is back to being the main repo.

**Comment by amitaibu** [comment]
Merged, thanks.

**Pull request merged**